scripts/core/CONTRIB.rst

```diff
@@ -41,103 +32,24 @@ accepted into the project.
     If the feature in question matches any of these criteria, please refer to
     the `Experimental Features`_ section, otherwise refer to the `Core
     Features`_ section. If you are unsure how to proceed please `create an
-    issue <https://github.com/oneapi-src/unified-runtime/issues/new>`_ asking
-    for clarification.
+    issue <https://github.com/intel/llvm/issues/new?template=Blank+issue>`_
```

Comment:
when you go to create a new issue on intel/llvm you get a list of issue templates to choose from, do we want an issue template for UR?

Reply:
I've added this question to my post-move todo list.